src/frontend/planner_test/tests/testdata/union.yaml

```diff
@@ -206,70 +206,22 @@
-    LogicalUnion { all: true }
-    ├─LogicalProject { exprs: [1:Int32] }
-    | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-    └─LogicalProject { exprs: [1:Int32] }
-      └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
+    LogicalValues { rows: [[1:Int32], [1:Int32]], schema: Schema { fields: [1:Int32:Int32] } }
   batch_plan: |
-    BatchUnion { all: true }
-    ├─BatchProject { exprs: [1:Int32] }
-    | └─BatchValues { rows: [[]] }
-    └─BatchProject { exprs: [1:Int32] }
-      └─BatchValues { rows: [[]] }
+    BatchValues { rows: [[1:Int32], [1:Int32]] }
 - sql: |
     select 1 union all select 2 union all select 3 union all select 4 union all select 5
   optimized_logical_plan: |
-    LogicalUnion { all: true }
-    ├─LogicalProject { exprs: [1:Int32] }
-    | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-    ├─LogicalProject { exprs: [2:Int32] }
-    | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-    ├─LogicalProject { exprs: [3:Int32] }
-    | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-    ├─LogicalProject { exprs: [4:Int32] }
-    | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-    └─LogicalProject { exprs: [5:Int32] }
-      └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
+    LogicalValues { rows: [[1:Int32], [2:Int32], [3:Int32], [4:Int32], [5:Int32]], schema: Schema { fields: [1:Int32:Int32] } }
   batch_plan: |
-    BatchUnion { all: true }
-    ├─BatchProject { exprs: [1:Int32] }
-    | └─BatchValues { rows: [[]] }
-    ├─BatchProject { exprs: [2:Int32] }
-    | └─BatchValues { rows: [[]] }
-    ├─BatchProject { exprs: [3:Int32] }
-    | └─BatchValues { rows: [[]] }
-    ├─BatchProject { exprs: [4:Int32] }
-    | └─BatchValues { rows: [[]] }
-    └─BatchProject { exprs: [5:Int32] }
-      └─BatchValues { rows: [[]] }
+    BatchValues { rows: [[1:Int32], [2:Int32], [3:Int32], [4:Int32], [5:Int32]] }
 - sql: |
-    select 1 union select 2 union select 3 union select 4 union select 5
+    select 1 union select 2 union select 3 union select 4 union select 5 union select 5
   optimized_logical_plan: |
     LogicalAgg { group_key: [1:Int32], aggs: [] }
-    └─LogicalUnion { all: true }
-      ├─LogicalProject { exprs: [1:Int32] }
-      | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-      ├─LogicalProject { exprs: [2:Int32] }
-      | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-      ├─LogicalProject { exprs: [3:Int32] }
-      | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-      ├─LogicalProject { exprs: [4:Int32] }
-      | └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
-      └─LogicalProject { exprs: [5:Int32] }
-        └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
+    └─LogicalValues { rows: [[1:Int32], [2:Int32], [3:Int32], [4:Int32], [5:Int32], [5:Int32]], schema: Schema { fields: [1:Int32:Int32] } }
```

Comment:
The schema: `fields: [1:Int32:Int32]` looks wierd 🤔

Reply:
Yes, it's due to project. I decided not to change. Discussed with tianshuo on this.